### PR TITLE
AWS - Update the credentials

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4586)

## Description
Added credentials for the *Access Key* and *Secret Key*.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by amshamah419** [comment]
Even though we aren't breaking BC, I'm wondering if we should include a BC notice to tell customers that the credentials should be migrated to the credential store as a better security practice. @glicht WDYT?

**Comment by glicht** [comment]
> Even though we aren't breaking BC, I'm wondering if we should include a BC notice to tell customers that the credentials should be migrated to the credential store as a better security practice. @glicht WDYT?

if we are not breaking, then no need.

File: Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py [hunk 1]
```diff
@@ -30,8 +40,7 @@ def __init__(self, aws_default_region, aws_role_arn, aws_role_session_name, aws_
         self.aws_role_session_duration = aws_role_session_duration
         self.aws_role_policy = aws_role_policy
         self.aws_access_key_id = aws_access_key_id
-        self.aws_secret_access_key = aws_secret_access_key
-        self.aws_session_token = aws_session_token
+        self.aws_secret_access_key, self.aws_session_token = extract_session_from_access(aws_secret_access_key, aws_session_token)
```
Comment (glicht) [inline Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py:43]: I think we need to fix `aws_session` function. It looks like the code won't go into the part to log in with access_key + session_token. See: https://github.com/demisto/content/blob/f24dbfef8a47c6cc2f6665dfb89dd3438af410be/Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py#L137

Comment (mmhw) [reply Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule.py:43]: I fixed it

**Review by glicht — APPROVED** [review]
Looks good

File: Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule_test.py [hunk 2]
```diff
@@ -176,3 +176,29 @@ def test_AWSClient_without_session_token():
...
+                         [
+                             ('access_key@@@session_token', None, ('access_key', 'session_token')),
+                             ('test1', None, ('test1', None)),
+                             ('test1', 'test2', ('test1', 'test2')),
+                             ('test1@@@test2', 'test3', ('test1@@@test2', 'test3')),
+                             ('', None, ('', None)),
+                             (None, '', (None, '')),
+                             (None, None, (None, None))
+                         ])
+def test_extract_session_from_access(access_key, session_token, expected):
```
Comment (bziser) [inline Packs/ApiModules/Scripts/AWSApiModule/AWSApiModule_test.py:191]: Great work!! :)

**Review by bziser — APPROVED** [review]

**Pull request merged**